src/utils/firebaseUtils.js

```diff
@@ -26,6 +27,12 @@ export const addProduct = ({
     createdAt: serverTimestamp(),
   });
 
+export const deleteProduct = async (productID, listToken = '') => {
+  const productRef = doc(db, listToken, productID);
+
+  return deleteDoc(productRef);
+};
+
```

Comment:
Your `deleteProduct` function is clear, simple, and powerful 💯 ! Take care with it 😄 Good work!